### DISCUSSION TITLE
I accidentely deleted too much files and files that are in write protected folders - v.4.6.0-win10-portable

### DISCUSSION THREAD

**Discussion by cxbignekoc** [bleachbit/bleachbit#1701] [Q&A]
Bleachbit has accidentally deleted a lot of files i dont want to erase.
I try to hold back my tears... (it's been a while since my last backup - but i got it running and will see whats there)

win10
BB 4.6.0 portable
winapp2.ini

BB touched mostly Folders and Files from this write protected Folder:

**C:\Users\username\Documents**

Some Folders are left but are mostly empty.

What is **not touched**:

**C:\Users\username\AppData\Roaming**

- What selection in BB prevents erasing write protected files in this folder, except mark this folders they should be untouched?

---

Similar Post #1702 

I know this function and i had no issues before, after i installed the winapp2.ini from the Community to have more options.
So somewhere i use bb the wrong way or set up a wrong checkmark.

How to find the Issue? (except the issue right in front of the computer)

- if anyone spotted these Folder Path in Results of BleachBit please comment. I could not found it yet.

**Comment by az0** [comment]
One way to troubleshoot is to select one group (e.g., Google Chrome) or individual options (e.g., Google Chrome - History) and run a preview for each one. Then search the preview for the paths that are affected.

Tip: You can copy the BleachBit log to a text editor like Notepad, and then use the search in Notepad.

If you do find any option that would delete c:\users\username\documents, please file an issue (bug) ticket here in GitHub. There should not be any options like this.

**Comment by dazu89** [comment]
I am having the same issue, did not trust my eyes at first. Filed a bug report. I am sorry for your data loss, @cxbignekoc. Where can I find the log list of deleted files?

**Comment by dazu89** [comment]
In my case, I had not activated community maintained cleaners, i.e. winapp2.ini.

**Reply by cxbignekoc** [reply]
I could recover some stuff. But some Folders are still empty what is a Windows Backup issue.

I have not found the Setting what cause this erase issue yet.
i found the most by BB deleted files in the $recycle.bin - List from BleachBit (Deep Search?)
in my case its JSON-Preset Files and Folders.

I saw that BleachBit erased *.json files from another Apps (in my case ShareX) in the same Folder.
Its deleting also complete write protected Folders (what not means delete protected?!) like:

- C:\Users\username\Documents\ExpressiveE
- C:\Users\username\Documents\gr-8
- C:\Users\username\Documents\iZotope
- C:\Users\cylob\Documents\Sonic Charge
- C:\Users\cylob\Documents\Vital
for example.

**Comment by az0** [comment]
I created an issue and am closing this as a discussion. Please see https://github.com/bleachbit/bleachbit/issues/1789
